packages/console/src/components/Button/index.tsx

```diff
@@ -51,6 +51,7 @@ function Button({
   loadingDelay = 500,
   onClick,
   trailingIcon,
+  href,
```

Comment:
Find a better name?

Reply:
Renamed as `previewLink` to avoid using pre-defined props. Add LOG-5866 to track the upcoming refactor.